### PR TITLE
Docker/Kubernetes events collector

### PR DESCRIPTION
**Pull request by rabbitstack**
127 0 0 1_8000_agents_sematext-agent_containers_events_

127 0 0 1_8000_agents_sematext-agent_kubernetes_events_

### CODE REVIEW

**Comment by adnanrahic** [comment]
@rabbitstack what's our next step here?

**Comment by rabbitstack** [comment]
@adnanrahic I've renamed the events excluded flags and they will be available in the next release. Feel free to review now.

**Comment by rabbitstack** [comment]
Published agent 1.2.3

**Comment by rabbitstack** [comment]
ping @adnanrahic

File: docs/agents/sematext-agent/containers/events.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+title: Container events
+description: Explains how to control the event stream collector and available filters
+
+Sematext Agent collects all sorts of container life-cycle, image, daemon, volume, plugin, and network events. If you're not interested in a certain event category, you can instruct the agent to ignore events that pertain to such a category via the `CONTAINER_EXCLUDED_EVENTS` environment variable. For example, to drop the volume and image events you would define the following environment variable:
```
Comment (adnanrahic) [inline docs/agents/sematext-agent/containers/events.md:4]: Should be `CONTAINER_EVENTS_EXCLUDED` ????

File: docs/agents/sematext-agent/kubernetes/events.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+- PersistentVolume
+- ServiceAccount
+- Role
+- Binding
+- ClusterRole
+- ClusterRoleBinding
+
+### Namespace filter
+
+Sematext Agent collects events from all existing namespaces. If you want to collect events from the specific namespace, set the `KUBERNETES_EVENTS_NAMESPACE` environment variable to desired Kubernetes namespace.
```
Comment (adnanrahic) [inline docs/agents/sematext-agent/kubernetes/events.md:43]: If you want to collect events from the specific namespace, set the `KUBERNETES_EVENTS_NAMESPACE` environment variable to desired Kubernetes namespace.

=>

If you want to collect events from a specific namespace, set the `KUBERNETES_EVENTS_NAMESPACE` environment variable to the desired Kubernetes namespace.

File: docs/agents/sematext-agent/containers/events.md (hunk 1 above)
Comment (rabbitstack) [reply docs/agents/sematext-agent/containers/events.md:4]: Yes, good catch :)

**Pull request merged**